src/environment/dev/CampaignParameterOverride.ts

```diff
@@ -15,7 +15,7 @@ import { CampaignParameters } from '@src/domain/CampaignParameters';
 export function getCampaignParameterOverride( campaignParameters: CampaignParameters ): CampaignParameters {
 	return {
 		...campaignParameters,
-		startDate: '2023-10-30',
-		isLateProgress: true
```

Comment:
when I checked this out locally I had to overwrite `endDate` as well, because I think for me it was still a 2023 date and that caused a local error -> banner was not loading.

not sure if you want to add it here for further development purposes (until the real CampaignParameters2024 are updated properly)

Reply:
Thanks!